diff --git a/node_modules/@react-native-async-storage/async-storage/android/build.gradle b/node_modules/@react-native-async-storage/async-storage/android/build.gradle
index 4f80202..a7ac653 100644
--- a/node_modules/@react-native-async-storage/async-storage/android/build.gradle
+++ b/node_modules/@react-native-async-storage/async-storage/android/build.gradle
@@ -78,6 +78,9 @@ android {
 }
 
 repositories {
+    maven {
+        url("$projectDir/local_repo")
+    }
     mavenCentral()
     google()
 }
